Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM php:7.4-buster
+
+LABEL maintainer="Couchbase"
+
+WORKDIR /app
+
+ADD . /app
+
+# Install dependencies
+RUN apt-get update -y && apt-get install -y \
+    software-properties-common \
+    git-all \
+    gnupg2 wget vim \
```

Comment:
vim?!

Reply:
Was messing around inside the container to test some things. Probably useful, but I can remove it as we don't include it in others.